notebooks/idr0062_pixel_classification_zarr.ipynb

```diff
@@ -727,6 +727,13 @@
     "compare(first_channel, data[0, 0, :, :, :], shadow=False, gradient_opacity=0.2, ui_collapsed=True)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![Compare](images/ilastik_1.png)"
```

Comment:
I cannot see that screenshot in the rendered notebook. Is that maybe not added ? Do not see that png added in the diff of the PR

Reply:
Sorry I forgot to push the images
Done